examples/terraform/Earthfile

```diff
@@ -0,0 +1,19 @@
+
+build:
```

Comment:
We should add this example to the root Earthfile, under the `+examples` target. That'll cause the CI to pick it up.

Reply:
Yep, realized I spaced that. Fixed.